security/4.0/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Jakarta Security 4.0 (under development)"
+date: 2023-05-30
+summary: "Release for Jakarta EE 11"
+---
+Jakarta Security defines a standard for creating secure Jakarta EE applications in modern application paradigms.
+
+The release splits focuses on evolving the API in various ways, and specifically providing APIs for the authorization theme (including interceptors and an abstraction for the permission store)
+
+### New features, enhancements or additions
+* Client-cert and Digest SECURITY #120 
+* Authentication mechanism per URL SECURIY #86 
+* User choice of authentication mechanism (login with provider X, login with provider Y, etc) 
+* Multiple authentication mechanisms (try JWT, fallback to BASIC, etc) 
+* @RolesAllowed alternative 
+* Easily adding an interceptor to a build-in CDI bean
```

Comment:
What does this mean?

Reply:
> Easily adding an interceptor to a build-in CDI bean

See this post for some more background: https://arjan-tijms.omnifaces.org/2017/08/dynamically-adding-interceptor-to-build.html

Also see the discussion here: https://github.com/jakartaee/cdi/issues/611

Reply:
@arjantijms the issue https://github.com/jakartaee/cdi/issues/611 was closed without any changes. I am not sure how you can achieve this feature in the Jakarta Security specification? Do you have an issue in the Jakarta Security spec to achieve the same goal?

Reply:
Sorry for the late reply, yes, there are two issues covering this:

* https://github.com/jakartaee/security/issues/192
* https://github.com/jakartaee/security/issues/200